include/image-commands.mk

```diff
@@ -175,6 +179,11 @@ define Build/check-size
 	}
 endef
 
+define Build/dlink-sge-image
```

Comment:
Hi @s-2 ,

Do you think this should belong to the other commit? It did work for me now by the way.

Reply:
Indeed this would be better off in the first commit, considering it is not within the target-specific makefile here. 

However I'll leave the signature stuff in the second commit, since this seems to be specific to ath79 devices, thus COVR-C1200 being the first actual device to introduce this format.
Not sure about append-file, but I guess the buildstep is way less complex for the ramips devices and does not need this either (since no OKLI loader is required).

Reply:
For the dir-853-a3 only the slink-sge-image was needed at the end of the build recipe and nothing else.

Hopefully others will confirm that this is working as well.